src/Moryx.Resources.Management/Facades/ResourceManagementFacade.cs

```diff
@@ -131,8 +131,6 @@ public TResult Read<TResult>(long id, Func<Resource, TResult> accessor)
             ValidateHealthState();
 
             var resource = ResourceGraph.Get(id);
-            if (resource == null)
-                throw new KeyNotFoundException($"No resource with Id {id} found!");
```

Comment:
Is this still consistent with the behavior of Get(long id)?

Reply:
The Graph also returns null and the Facade shoudn't change that